lib/commands/context.js

```diff
@@ -212,9 +212,10 @@ async function setupNewChromedriver (opts, curDeviceId, adbPort) {
     adbPort
   };
   let chromedriver = new Chromedriver(chromeArgs);
+  let appPackage = (opts.chromeOptions && opts.chromeOptions.androidPackage) || opts.appPackage;
```

Comment:
if you read the code below, it apparently also allows `chromeAndroidPackage` to be on `opts`; should we handle that case as well?